src/test/java/seedu/address/logic/commands/AddContactedCommandTest.java

```diff
@@ -52,40 +47,41 @@ public void execute_invalidIndex_throwsCommandException() {
         // ensures that outOfBoundIndex is out bounds of address book list
         assertTrue(outOfBoundIndex.getZeroBased() >= model.getAddressBook().getPersonList().size());
 
-        ContactedCommand contactedCommand = new ContactedCommand(
-                outOfBoundIndex, RecentDate.parse(VALID_CONTACTED_DATE_AMY), new Description(VALID_CONTACTED_DESC_AMY));
+        AddContactedCommand contactedCommand = new AddContactedCommand(
+                outOfBoundIndex, new ContactedInfo(VALID_CONTACTED_DATE_AMY, VALID_CONTACTED_DESC_AMY));
 
         assertCommandFailure(contactedCommand, model, Messages.MESSAGE_INVALID_PERSON_DISPLAYED_INDEX);
     }
 
     @Test
     public void execute_validIndex_success() {
-        showPersonAtIndex(model, INDEX_FIRST_PERSON);
-
-        ContactedCommand contactedCommand = new ContactedCommand(INDEX_FIRST_PERSON,
-                RecentDate.parse(VALID_CONTACTED_DATE_AMY), new Description(VALID_CONTACTED_DESC_AMY));
+        // showPersonAtIndex(model, INDEX_FIRST_PERSON);
```

Comment:
oh ya I forgot to un-comment.. But `showPersonAtIndex(model, INDEX_FIRST_PERSON);` needs to be there when you are testing filter list, instead of unfiltered list.